src/utils/wsRedirect.ts

```diff
@@ -84,7 +84,7 @@ export class WsRedirect {
   }
 
   createCredential (params: queryParams, credentials: string[]): void {
-    if (credentials != null) {
+    if (credentials?.[0] != null && credentials[1] != null) {
```

Comment:
this looks subject to an index out of bounds error, where is the length of the array checked?

Reply:
valid.  updated